lib/protobuf/active_record/transformer.rb

```diff
@@ -0,0 +1,22 @@
+module Protobuf
+  module ActiveRecord
+    class Transformer
+      attr_accessor :callable, :options
+
+      def initialize(callable, options = {})
+        @callable = callable
+        @options = options
+      end
+
+      def call(proto)
+        callable.call(proto)
+      end
+
+      def nullify?(proto)
+        return false unless options[:nullify_on]
+
+        proto.field?(:nullify) && proto.nullify.include?(options[:nullify_on].to_s)
```

Comment:
I think the correct way to do this is when `attribute_from_proto` is declared we should have access to the message class and be able to inspect it, and avoid inspecting each individual message all together.

I can add a warning for now though.